src/models/market_data_history.rs

```diff
@@ -0,0 +1,65 @@
+use chrono::naive::serde::ts_milliseconds;
+use chrono::NaiveDateTime;
+use serde::{Deserialize, Serialize};
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Bar {
+    pub o: f64,
+    pub c: f64,
+    pub h: f64,
+    pub l: f64,
+    pub v: i64,
+    #[serde(with = "ts_milliseconds")]
+    pub t: NaiveDateTime,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MarketDataHistory {
+    pub server_id: String,
+    pub symbol: String,
+    pub text: String,
+    pub price_factor: i64,
+    #[serde(with = "parse_datetime")]
+    pub start_time: NaiveDateTime,
+    #[serde(default)]
```

Comment:
you already impl `Default` for the whole struct, `#[serde(default)]` is not doing anything above these, also shouldn't the high and low be `rust_decimal::Decimal`?

Reply:
No high and low appear as strings like "123/456/789". I think we could derive a separate struct for them later.

```
string
High value during this time series with format %h/%v/%t. 
%h is the high price (scaled by priceFactor), 
%v is volume (volume factor will always be 100 (reported volume = actual volume/100)) and 
%t is minutes from start time of the chart
```